Paraswap-Agents/AdminOperations/src/agent.spec.ts

```diff
@@ -0,0 +1,240 @@
+import { BigNumber } from "ethers";
+import { Interface } from "ethers/lib/utils";
+import { Finding, FindingType, FindingSeverity, TransactionEvent, HandleTransaction } from "forta-agent";
+import { createAddress, TestTransactionEvent } from "forta-agent-tools/lib/tests";
+import { provideHandleTransaction } from "./agent";
+import { SWAPPER_IFACE } from "./utils";
+
+// AugustusSwapper address
+const SWAPPER_ADDR = createAddress("0xa1");
+// Address used to initiate transactions
+const USER_ADDR = createAddress("0xaa");
+// Irrelevent events/ functions used for tests
+const IRRELEVENT_IFACE = new Interface(["event wrongsig()", "function wrongFunction()"]);
+
+const TEST_DATA = [
+  [createAddress("0xb1"), createAddress("0xc1"), BigNumber.from(10)], // transferTokens args
+  [createAddress("0xb2"), createAddress("0xc2"), BigNumber.from(80)], // transferTokens args
+  [createAddress("0xb3"), createAddress("0xc3"), BigNumber.from(90)], // transferTokens args
+  [USER_ADDR, createAddress("0xc1"), BigNumber.from(10)], // Transfer event args
+  [USER_ADDR, createAddress("0xc2"), BigNumber.from(80)], // Transfer event args
+  ["0xafde1234", createAddress("0xd1")], // setImplementation args
+  [createAddress("0xd3")], // setFeeWallet args
+  [createAddress("0xd4")], // RouterInitialized args
+  [createAddress("0xd5")], // AdapterInitialized args
+];
+
+// function to generate test findings
+const createFinding = (operationName: string, args: any[]) => {
```

Comment:
Missing option to create a Finding with `alertId` of `PARASWAP-1-4` (`registerPartner`).